injected/src/features/broker-protection/types.js

```
@@ -12,6 +12,7 @@
  * @property {string} [captchaType]
  * @property {string} [injectCaptchaHandler]
  * @property {string} [dataSource]
+ * @property {boolean} [failSilently]
```

`failSilently` suggests we are allowing something to fail silently when in reality we're just skipping the action in case of failure, can we consider something on the lines of: `bypassOnFailure` | `bypassOnError` | `skipOnError`?